src/objectTranslations/Contact-nl_NL.objectTranslation

```diff
@@ -3481,11 +3485,11 @@
     <label>Geslacht/gender</label>
     <picklistValues>
       <masterLabel>Female</masterLabel>
-      <translation>Vrouw</translation>
+      <translation>Female</translation>
     </picklistValues>
     <picklistValues>
       <masterLabel>Male</masterLabel>
-      <translation>Man</translation>
+      <translation>Male</translation>
```

Comment:
These values should remain translated. Only the specific custom label should have had its translation reversed.
```suggestion
      <translation>Man</translation>
```